src/devices/QwiicButton/QwiicButton.ButtonStatus.cs

```diff
@@ -0,0 +1,68 @@
+﻿//// Licensed to the .NET Foundation under one or more agreements.
+//// The .NET Foundation licenses this file to you under the MIT license.
+//// See the LICENSE file in the project root for more information.
+
+using Iot.Device.QwiicButton.RegisterMapping;
+
+namespace Iot.Device.QwiicButton
+{
+    public sealed partial class QwiicButton
+    {
+        /// <summary>
+        /// Returns whether the button is pressed.
+        /// </summary>
+        public bool IsPressed()
+        {
+            var status = new StatusRegisterBitField(_registerAccess.ReadSingleRegister(Register.ButtonStatus));
+            return status.IsPressed;
+        }
+
+        /// <summary>
+        /// Returns whether the button is clicked.
+        /// </summary>
+        public bool HasBeenClicked()
+        {
+            var status = new StatusRegisterBitField(_registerAccess.ReadSingleRegister(Register.ButtonStatus));
+            return status.HasBeenClicked;
+        }
+
+        /// <summary>
+        /// Returns whether a new button status event has occurred.
+        /// </summary>
+        public bool EventAvailable()
```

Comment:
once you read this I assume you should call `ClearEventBits` immediatelly after, right? If so please make sure there is a remark in the xml doc about this

Reply:
Done